src/Work/Formulas.ts

```diff
@@ -119,7 +120,18 @@ export const calculateCompanyWorkStats = (
   const gains = scaleWorkStats(
     multWorkStats(
       {
-        money: companyPosition.baseSalary * company.salaryMultiplier * bn11Mult * currentNodeMults.CompanyWorkMoney,
+        money:
+          companyPosition.baseSalary *
+          company.salaryMultiplier *
+          bn11Mult *
+          currentNodeMults.CompanyWorkMoney *
+          (Player.augmentations
+            .map((aug: PlayerOwnedAugmentation) => {
+              return aug.name;
+            })
+            .filter((aug) => aug == AugmentationName.EmpathySuppressor).length == 1
+            ? Math.pow(10, Object.values(Player.jobs).filter((job: string) => job == "Chief Executive Officer").length)
```

Comment:
Am I reading this right that it's a 10x boost per CEO position, e.g. 3 CEO positions = 1000x salary?

My initial reaction is that this is probably overtuned. Certainly open to other feedback on this.

Reply:
You have to become a CEO somewhere, join Silhouette, buy the Aug, install, and then become a CEO again before you ever see any use from it. With the time investment involved and what you have to not be doing in order to get those CEO spots, the balancing seems about right, and it brings back company work as something work doing, *eventually*. But then you still have to gain rep to go about getting your augs and whatnot afterwards. I suspect that it may actually be too weak, but have been wracking my brain for months for *something* to give Silhouette, which is a pain in the butt to join, and has one weak unique aug and the rest are shared with other factions.

Reply:
10^number of CEO positions isn't just "makes company work worth doing eventually," it is just a total balance destroyer. If company work is ever worthwhile whatsoever, then being able to possibly get e.g. millions of times the return from it is just not in line with what augments do. There's no other augment that affects something with that kind of multiple, the highest is like a 5x total multiplier for a parameter. Even with just one CEO spot, this aug would be doubling the highest existing multiplier, and it scales exponentially from there (2->100x, 3->1000x).

A lategame aug that scales really well should not be out of the question, but to me 10^x just seems so far from how anything else in the game is balanced. And with it being locked behind a lategame setup, it does nothing to make the underlying mechanic fundamentally less weak for the normal portion of gameplay (if the problem is that company work is weak).

But I'd love to get feedback from some other folks, maybe I'm missing something here.

Reply:
I agree, it feels like a better solution would be to make Silhouette less awful to get into, and then it wouldn't feel weak to have a softer buff. Incredibly strong buffs are incredibly hard to balance, they're always on a knife's edge of either being useless or game breaking.

Reply:
Exponential in num CEO positions seems a bit too much, yeah. Maybe the boost could be something like calcMult(charisma) * numCEOpositions, where calcMult() is similar to calculateIntelligenceMult(). Would scale decently (which can be balanced in calcMult) and create some more use for charisma as a stat.